PIL/Image.py

```diff
@@ -667,6 +715,7 @@ def __setstate__(self, state):
         self.frombytes(data)
 
     def tobytes(self, encoder_name="raw", *args):
```

Comment:
@wiredfool as you can probably tell, I'm still new to this, and contributing to open source. I'm trying though.